accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/aceinstaller/AceBeanInstallerIncremental.java

```diff
@@ -97,6 +98,12 @@ protected void installAcl(Set<AceBean> aceBeanSetFromConfig, String path, Set<St
             if (!principalsInConfiguration.contains(acePrincipalName)) {
                 countOutsideConfig++;
                 diffLog.append("    OUTSIDE (not in Config) " + actualAceBeanCompareStr + "\n");
+                // Condition will check if everyone deny was added at the bottom, blocking all permissions created by actool
+                // usually happens when SP installs some rep:policy nodes
+                if ((acl.size() - countOutsideConfig <= configuredAceEntries.size())

```

Comment:
I don't understand how this condition checks the position of the everyone deny (what is stated in the comment), or is `everyoneDenyFound` always `true` irrespective of the position of the deny for everyone?